src/Functions/formatDateTime.cpp

```diff
@@ -80,10 +90,25 @@ template <> struct ActionaValueTypeMap<DataTypeDateTime64> { using ActionValueTy
   *
   * PS. We can make this function to return FixedString. Currently it returns String.
   */
-class FunctionFormatDateTime : public IFunction
+template <typename Name, bool support_integer>
```

Comment:
The lack of comments about "support_integer" makes code difficult to read.
I will add a comment after merge.